### PR TITLE
Rename Datastore helper to Helper and move to pkg

### PR DESCRIPTION
**Pull request by fdawg4l**
This struct is fairly useful in that it gives you a datastore object
relative to a given path.  And the methods attached to it do the URI
munging for the caller (relative to that root path).  Might as well move it to /pkg.

### CODE REVIEW

**Comment by dougm** [comment]
The name, 'datastore.DSWrapper' - `DS` stutters a bit, seems `datastore.Wrapper` would do?
We also have some methods named `FooHelper`, can we stick with just using `Helper` instead of also having `Wrapper`?

There's a number of these: `uuid.New().String()[0:16]`.  Some use a `-` delim, some don't.
Might be nice to have `datastore.ID(name string)` to concat the random bits + delim + name.

**Comment by fdawg4l** [comment]
Struggled with the name.  Will rename to `Helper`.    Will also add a utility function to build the name.  It's for tests only which happen to live in other packages.

**Comment by fdawg4l** [comment]
Updated.  Fire when read.  Thanks!

File: lib/portlayer/storage/vsphere/parent_test.go [hunk 1]
```diff
@@ -101,3 +109,49 @@ func TestParentSaveRestore(t *testing.T) {
 		return
 	}
 }
+
+func Session(ctx context.Context, t *testing.T) *session.Session {
```
Comment (dougm) [inline lib/portlayer/storage/vsphere/parent_test.go]: Looks like this and DSsetup below are not used?  Tests above are using datastore.DSsetup, which looks about the same.

File: pkg/vsphere/datastore/datastore_util.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	}
+
+	return s
+}
+
+func DSsetup(t *testing.T) (context.Context, *Helper, func()) {
+	ctx := context.Background()
+	sess := Session(ctx, t)
+
+	ds, err := NewHelper(ctx, sess, sess.Datastore, uuid.New().String()[0:16]+"dstests")
```
Comment (dougm) [inline pkg/vsphere/datastore/datastore_util.go]: Looks like you could use TestName("dstests") here instead

File: lib/portlayer/storage/vsphere/parent_test.go (hunk 1 above)
Comment (fdawg4l) [inline lib/portlayer/storage/vsphere/parent_test.go]: Oops.  Done.

File: pkg/vsphere/datastore/datastore_util.go (hunk 2 above)
Comment (fdawg4l) [inline pkg/vsphere/datastore/datastore_util.go]: Done.

**Comment by dougm** [comment]
lgtm

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

Do you need to change github PR title as it still refers the old name?

**Comment by fdawg4l** [comment]
Thanks!  Updated title.  Will squash too!

**Pull request merged**